src/Tes.Runner/Storage/TerraUrlTransformationStrategy.cs

```diff
@@ -16,30 +17,39 @@ namespace Tes.Runner.Storage
 {
     public class TerraUrlTransformationStrategy : IUrlTransformationStrategy
     {
+        public const int TokenExpirationInSeconds = 3600 * 24; //1 day, max time allowed by Terra. 
+        public const int CacheExpirationInSeconds = TokenExpirationInSeconds - 1800; // 30 minutes less than token expiration
```

Comment:
TimeSpan.FromMinutes(30) easier to read and doesn't require comment documentation